src/types/onyx/PersonalDetails.ts

```diff
@@ -35,10 +35,10 @@ type PersonalDetails = {
     /** Timezone of the current user from their personal details */
     timezone?: {
         /** Value of selected timezone */
-        selected?: string;
+        selected: string;
 
         /** Whether timezone is automatically set */
-        automatic?: boolean;
+        automatic: boolean;
     };
```

Comment:
Let's keep those ones as optional, in [this PR](https://github.com/Expensify/App/issues/24909) they can be optional.